pallets/staking/src/lib.rs

```diff
@@ -3798,7 +3862,8 @@ where
         slash_fraction: &[Perbill],
         slash_session: SessionIndex,
     ) -> Result<Weight, ()> {
-        if !Self::can_report() {
+        // No need to run through when Slashing is off.
+        if !Self::can_report() || Self::slashing_status() == SlashingSwitch::Off {
```

Comment:
when slashing is off, we should be submitting `Ok`, not `Err`. i.e. Offenses should still be reported but no slashing should happen of offenses. See L3881 for an example.

Reply:
Hmm, I thought I fix this in my local code 🤔